h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -611,6 +626,8 @@ public final double linkInvDeriv2(double x) {
           return 0;
         case log:
           return Math.max(x, Double.MIN_NORMAL);
+        case tweedie:
+          return _link_power==0?Math.max(x, Double.MIN_NORMAL):x*_oneOLinkPower*(_oneOLinkPower-1)*_oneOetaSquare;
         default:
```

Comment:
The link function depends on the link power q as follows:

image

I am just implementing this.